nix/default.nix

```diff
@@ -45,8 +45,8 @@ import sources.nixpkgs {
       gaiad = pkgs.callPackage ./bin.nix {
         appName = "gaiad";
         version = "v11.0.0";
-        binUrl = "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-linux-amd64";
-        sha256 = "sha256-JY3y7sWyL4uq3JiOGE+/0q5vn4iOn0RhoRDMNl/oYwA=";
+        binUrl = "https://github.com/cosmos/gaia/releases/download/v11.0.0/gaiad-v11.0.0-darwin-arm64";
+        sha256 = "sha256-U9D/5Ng1PlHQvlQ+33ZN4DPiTXA9TECCRKFB5jWydig=";
```

Comment:
This is part of the original PR @facs95 branched from which is the nix PR. By the time that's ready it will be ported back to linux. I've only changed it to darwin so I can run it locally.